examples/inferencing/running-on-gke.md

```diff
@@ -0,0 +1,185 @@
+## Deployment
```

Comment:
Please add an introduction section describing what does these instructions do. Mention what models are supported, what would be spun up after the deployment and what it can be used for.